scenes/passy_scene_read_error.c

```diff
@@ -21,6 +21,11 @@ void passy_scene_read_error_on_enter(void* context) {
     FuriString* primary_str = furi_string_alloc_set("Read Errror");
     FuriString* secondary_str = furi_string_alloc_set("Try again?");
 
+    if(passy->read_type == PassyReadDG7) {
```

Comment:
We can leave this, but it's not technically correct.  There are many reasons for the failure, like the user putting in bad auth data.